code/modules/research/designs/mechfab_designs.dm

```diff
@@ -862,4 +862,79 @@
 	id = "cooling_module"
 	req_tech = list(TECH_MATERIAL = 2, TECH_POWER = 3, TECH_ENGINEERING = 3)
 	materials = list(DEFAULT_WALL_MATERIAL= 7000, "glass"= 5500)
-	build_path = /obj/item/rig_module/cooling_unit
\ No newline at end of file
+	build_path = /obj/item/rig_module/cooling_unit
+
+
+/// RIGS
+
+/datum/design/rig
+	build_type = MECHFAB
+	category = "RIG Assemblies"
+	materials = list(DEFAULT_WALL_MATERIAL = 25000, "glass" = 10000)
+	req_tech = list(TECH_MATERIAL = 3, TECH_ENGINEERING = 2, TECH_MAGNET = 3, TECH_POWER = 3)
+	time = 20
+
+/datum/design/rig/ce
+	name = "advanced voidsuit control module assembly"
+	desc = "An assembly frame for an advanced voidsuit that protects against hazardous, low pressure environments."
+	id = "rig_ce"
+	build_path = /obj/item/rig_assembly/ce
+
+/datum/design/rig/eva
+	name = "EVA suit control module assembly"
+	desc = "An assembly for light rig that is desiged for repairs and maintenance to the outside of habitats and vessels."
+	id = "rig_eva"
+	build_path = /obj/item/rig_assembly/eva
+
+/datum/design/rig/industrial
+	name = "industrial suit control module assembly"
+	desc = "An assembly for a heavy, powerful rig used by construction crews and mining corporations."
+	id = "rig_industrial"
+	materials = list(DEFAULT_WALL_MATERIAL = 30000, "glass" = 12500)
+	build_path = /obj/item/rig_assembly/industrial
+
+/datum/design/rig/hazmat
+	name = "AMI control module assembly"
+	desc = "An assembly for Anomalous Material Interaction hardsuit that protects against the strangest energies the universe can throw at it."
+	id = "rig_hazmat"
+	build_path = /obj/item/rig_assembly/hazmat
+
+/datum/design/rig/medical
+	name = "rescue suit control module assembly"
+	desc = "An assembly for a durable suit designed for medical rescue in high risk areas."
+	id = "rig_medical"
+	build_path = /obj/item/rig_assembly/medical
+
+/datum/design/rig/hazard
+	name = "hazard hardsuit control module"
+	desc = "An assembly for a security hardsuit designed for prolonged EVA in dangerous environments."
+	id = "rig_hazard"
+	build_path = /obj/item/rig_assembly/combat/hazard
+	req_tech = list(TECH_MATERIAL = 4, TECH_ENGINEERING = 3, TECH_MAGNET = 3, TECH_POWER = 3, TECH_COMBAT = 3)
+	materials = list(DEFAULT_WALL_MATERIAL = 45000, "glass" = 20000)
+
+/datum/design/rig/combat
```

Comment:
Why are these legal? They have:
```
	initial_modules = list(
		/obj/item/rig_module/mounted,
		/obj/item/rig_module/vision/thermal,
		/obj/item/rig_module/grenade_launcher,
		/obj/item/rig_module/ai_container,
		/obj/item/rig_module/power_sink,
		/obj/item/rig_module/electrowarfare_suite,
		/obj/item/rig_module/chem_dispenser/combat
		)
```
Aren't they incredibly OP to be produced by the station?

Reply:
Uhm, that is for equipped version only. RND produces vanilla, equipped version is only use for adminbus spawn. Also if you noticed I nerfed armour values to more appropriate level where it is not as good as say ERT or Mercs RIGs